cmd/api/prometheus.go

```diff
@@ -5,9 +5,19 @@ import (
 
 	"github.com/Chatterino/api/pkg/config"
 	"github.com/go-chi/chi/v5"
+	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/client_golang/prometheus/promhttp"
 )
 
+var (
+	servedRoutes = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Name: "requests_total",
+		Help: "Total number of requests",
+	},
+		[]string{"path"},
+	)
+)
```

Comment:
This doesn't seem to be used, can this be removed?

Reply:
that is indeed unneeded